uint32 height
uint32 width
uint32 camera_fps
uint32 imu_frequency
string distortion_model
float64[16] TSC_c0
float64[2] focal_length_c0
float64[2] principal_point_c0

float64[4] D_c0
float64[9] K_c0
float64[9] R_c0
float64[12] P_c0

float64[16] TSC_c1
float64[2] focal_length_c1
float64[2] principal_point_c1

float64[4] D_c1
float64[9] K_c1
float64[9] R_c1
float64[12] P_c1

float64 imu_sigma_g_c
float64 imu_sigma_a_c
float64 imu_sigma_bg
float64 imu_sigma_ba
float64 imu_sigma_gw_c
float64 imu_sigma_aw_c
